app/models/setting.rb

```diff
@@ -83,4 +88,32 @@ def self.invoice_reminder_in_days
   def self.wishlist_size
     Setting.find_by(code: :wishlist_size).value.to_i
   end
+
+  def self.violations_count_regulations_link
+    hash = Setting.find_by(code: :violations_count_regulations_link)&.value
+    hash.present? ? JSON.parse(hash).with_indifferent_access[I18n.locale] : nil
+  end
+
+  private
+
+  def validate_value
+    methods = FORMAT_VALIDATIONS[code]
+    return unless methods
+
+    methods.each do |method|
+      send(method)
+    end
+  end
+
+  def validate_json_hash
+    return if parsable_json_hash?(value)
+
+    errors.add(:value, :invalid)
```

Comment:
It should be `errors.add(:value, :invalid) unless parsable_json_hash?(value)`. This is the mainline of code, it should not be a guard clause.